lib/puppet/type/cron.rb

```diff
@@ -110,30 +110,11 @@ def alphacheck(value, ary)
     end
 
     def should_to_s(newvalue = @should)
-      if newvalue
-        newvalue = [newvalue] unless newvalue.is_a?(Array)
-        if self.name == :command or newvalue[0].is_a? Symbol
```

Comment:
`format_value_for_display` doesn't know that `:command` needs to be handled specially, so isn't this a regression?